exampleCourse/questions/demo/surveyQuestion/question.html

```diff
@@ -0,0 +1,48 @@
+<pl-question-panel>
+
+        <p>Please answer the following demographic survey questions. </p>
+        <p>As this is a survey, there are no correct or incorrect responses.</p>
+
+        <hr>
+
+        <h6>Race/Ethnicity: Check all that apply</h6>
```

Comment:
Nit: this is an inappropriate usage of `<h6>`, as it doesn't follow the proper hierarchy (that is, there's no `<h5>`/`<h4>`/etc. above it in the DOM).

Reply:
Should this just be in a `<p>` tag then?

Reply:
Done here https://github.com/PrairieLearn/PrairieLearn/pull/1867/commits/1fe5f5fe4ef56d022cb77a461a1d73e7ab80596e